gbenchmarks/bench/test1.cc

```diff
@@ -0,0 +1,30 @@
+#include <benchmark/benchmark.h>
+#include <cmath>
+#include <VecCore/VecCore>
+
+using namespace vecCore;
+
+static void LIBMVEC_Exp(benchmark::State &state)
+{
+  float *input;
+  input = (float *)AlignedAlloc(VECCORE_SIMD_ALIGN, 8192 * sizeof(float));
+  using Float_v = backend::Scalar::Float_v;
+
+  for(int i = 0; i < 8192; i++)
+  {
+    input[i] = 0.123f;
+  }
+
+  while (state.KeepRunning())
+  {
+      for(size_t i = 0; i < 8192; i += VectorSize<Float_v>())
+      {
+        math::Exp((Float_v &)(input[i]));
```

Comment:
@amadio 
For preliminary benchmarking purposes : 

When getting run time for normal case(that is without libmvec).
- `ifdef` is changed to `ifndef` in  `#ifdef VECCORE_ENABLE_LIBMVEC` ( VecMath.h ).
- The old Cube Root function is added back in VecMath.h, with `VECMATH_REAL_FUNC(Cbrt, cbrt)` commented out.
- `i += VectorSize<Float_v>()` is changed to `i++`

Unfortunately, there is massive slowdown when benchmarking the new `exp` function. Could this be due to the re-interpretation(using `(Float_v &)` per SIMD vector length) ?

Reply:
Hi Arif, not sure what you mean. I will try to run this locally and check on the performance.

Reply:
It looks like your commit unfortunately causes the performance degradation. For the scalar backend, there is no problem with vectorization with your commit or without it if I use `-ffast-math` to link libmvec functions (I tried with `std::sin()`). OpenMP is not needed at all.

Here are some results changing `std::exp` to `std::sin` and using `Scalar`, `Vc`, and `UMESimd` backends:

Without your commit and with `-O3` compilation flags:
```
-----------------------------------------------------
Benchmark              Time           CPU Iterations
-----------------------------------------------------
Scalar_Sin/1       46910 ns      46893 ns      15070
Vc_Sin/1           12498 ns      12493 ns      56160
UMESimd_Sin/1      12455 ns      12451 ns      56171
```
With `-O2 -ffast-math` and without your commit:
```
-----------------------------------------------------
Benchmark              Time           CPU Iterations
-----------------------------------------------------
Scalar_Sin/1        4320 ns       4319 ns     135581
Vc_Sin/1           11939 ns      11935 ns      58738
UMESimd_Sin/1      11923 ns      11919 ns      58712
```

With your commit and `-O2 -ffast-math` compilation flags:
```
-----------------------------------------------------
Benchmark              Time           CPU Iterations
-----------------------------------------------------
Scalar_Sin/1        4332 ns       4330 ns     158663
Vc_Sin/1           61142 ns      61122 ns      11497
UMESimd_Sin/1      61426 ns      61407 ns      11519
```

What we need is to get the timing we get for the `Scalar` backend with `-ffast-math` for all backends. I think that the best approach will be to create all benckmarks first, then use the results to make the changes to VecCore.